README.md

```diff
@@ -28,27 +28,14 @@ Now in Kolibri you should be able to navigate to a Explore tab.
 
 ## Setup
 
-Either if you want to build or develop the plugin, please follow these
-instructions.
-
-1. Install dependencies:
-
-```
-pip install -r requirements.txt --upgrade
-```
-
-2. Install node and yarn. Assuming you have nodeenv:
+Either if you want to build or develop the plugin, please run:
 
 ```
-nodeenv -p --node=10.17.0
-npm install -g yarn@1.22.10
+./scripts/bootstrap.sh
 ```
 
-3. Install the Javascript dependencies:
-
-```
-yarn install
-```
+Just like kolibri, we use a Python virtual environment along with Node
+to obtain the exact same dependencies.
 
 ## Building
 ### Bundling custom channel presentations
```

Comment:
The section "Using real content while developing a custom channel presentation" is left outdated here and could do with some edits.